### DISCUSSION TITLE
Set-PnPListItem Author and Editor update

### DISCUSSION THREAD

**Discussion by stephaneacknin-bib** [pnp/powershell#1075] [Q&A] [answered]
Hello,

Even if this subject has already been discussed, I didn't find out a solution.
I'm using powershell 5.1 with Pnp.Powershell version 1.7.0.

My goal is to update the Editor and the Author of an document in a specific Sharepoint site.
I 've tested lot of things but without success. The latest one is:
Set-PnPListItem -List $LibraryName -Identity $ItemID -Values @{"Author" = $NewOwner; "Editor" = $NewOwner} -UpdateType UpdateOverwriteVersion

The fields are nevr updated

Can you help me please?

Thanks

**Comment by AndersRask** [comment, marked as answer]
I tried the code on a dev tenant both with only Author and Author and Editor, and it works fine.

What are the value in $NewOwner variable? Is it an email on a valid user?
Does it work if you update other properties?
Where did you check the values? In the browser you need to check the values of "Modified By" and "Created By"

**Reply by stephaneacknin-bib** [reply]
Hello AndersRask,

thanks for your quick answer
- The $NewOwner variable is an email address that exists. I've tried with a string containing the email address ("xxx@domain.onmicrosoft.com") and also the object of the user retrieve from the AzureAd with its attribute Mail.
- I will try with another property. 
- I check the values in the Sharepoint site and also in the code by using a Get-PnPListItem -List $LibraryName -Id $ItemID after the set.

What are the versions of powershell and pnp.Powershell library you've used please?
I'm using a Powershell 5.1 and a Pnp.Powershell 1.7.0 to do my tests

thanks

**Reply by stephaneacknin-bib** [reply]
I've found the problem. 
It was due to the fact that I was not admin of the Sharepoint site
Sorry and thanks again for your help

**Reply by AndersRask** [reply]
good thing that you found the issue :)
